torchtune/utils/__init__.py

```diff
@@ -51,9 +41,15 @@
     "lora_fsdp_wrap_policy",
     "get_full_finetune_fsdp_wrap_policy",
     "get_unmasked_sequence_lengths",
-    "set_activation_checkpointing",
     "set_seed",
     "torch_version_ge",
+    "DEFAULT_PROFILE_DIR",
+    "DEFAULT_PROFILER_ACTIVITIES",
+    "DEFAULT_SCHEDULE",
+    "DEFAULT_TRACE_OPTS",
+    "DummyProfiler",
+    "PROFILER_KEY",
+    "setup_torch_profiler",
```

Comment:
do you mind double checking those? i dont understand what is being added here or why

Reply:
ah good catch, these should be removed